payjoin/src/receive/multiparty/error.rs

```diff
@@ -20,6 +24,30 @@ pub(crate) enum InternalMultipartyError {
     FailedToCombinePsbts(bitcoin::psbt::Error),
 }
 
+#[derive(Debug)]
+pub enum IdenticalProposalError {
+    IdenticalPsbts(Box<bitcoin::Psbt>, Box<bitcoin::Psbt>),
+    IdenticalContexts(Box<v2::UncheckedProposal>, Box<v2::UncheckedProposal>),
```

Comment:
Is it possible to just pass in the short id when constructing this error variant? I dont think we need the entire unchecked proposal

Reply:
Ok, moved it over to the mod so just the ShortId is passed through the error